docs/corpora_schema.md

```diff
@@ -185,17 +184,11 @@ The color code at the nth position in the array corresponds to category n in the
   </tr>
 </table>
 
-
-## **Implementations**
-
-The Data Portal requires submitted count matrices and associated metadata to be in one of three formats: AnnData, Loom, or a Seurat v3 RDS. Other formats are rejected.
-Each of these formats has a way to include metadata along with the count data, so a submission can be entirely contained within a single file. 
-
-
 ### **Scanpy/AnnData**
 
-[Anndata](https://anndata.readthedocs.io/en/stable/) is the hdf5-based file format used by scanpy. There is a python library for interacting with it. The
-count data is stored in an attribute `X` of shape (# of cells, # of genes). `X` can either be a numpy.ndarray or a scipy.sparse.spmatrix.
+The Data Portal requires submitted count matrices and associated metadata to be in [Anndata](https://anndata.readthedocs.io/en/stable/) format, the hdf5-based
+file format used by scanpy. There is a python library for interacting with it. The count data is stored in an attribute `X` of shape (# of cells, # of genes).
```

Comment:
```suggestion
file format used by scanpy — a python library for interacting with it. The count data is stored in an attribute `X` of shape (# of cells, # of genes).
```

Reply:
Scanpy is not properly a python library for interacting with AnnData, so I think this edit would be misleading. The text in this paragraph is a bit awkward, but it wasn't written in this PR and it'll be changed if we integrate the encoding, so I'd prefer to leave editorial modifications for the next pass.